storage/wavefront.go

```diff
@@ -41,9 +41,14 @@ func Wavefront(userInput map[string]string, wg *sync.WaitGroup) {
 
 	if proxyHost != "" {
 		Debug(userInput, fmt.Sprintf("Using proxy host: %s", proxyHost))
+		proxyMetricPort, err := strconv.Atoi(userInput["storage_driver_wf_metric_port"])
```

Comment:
Can you check if it is empty and default to 2878. Otherwise if parsing fails log a fatal error message.